app/services/stories_api/v3/endpoints/story.rb

```diff
@@ -58,6 +64,12 @@ def delete
 
           create_response(status: 204)
         end
+
+        def strip_admin_params
+          [:approved, :featured].map do |field|
```

Comment:
I think you can use `each` in this scenario instead of `map` since you're not returning the result.